core/src/main/java/org/elasticsearch/search/suggest/Suggest.java

```diff
@@ -553,7 +558,7 @@ public Option(Text text, Text highlighted, float score) {
                 }
 
                 public Option(Text text, float score) {
-                    this(text, null, score);
+                    this(text, null, score, null);
```

Comment:
Nothing wrong, just the collateMatch default value (`null`) when it's missing is defined at 2 places instead of one